values.yaml

```diff
@@ -52,5 +52,21 @@ configs:
       - tuf
       - tuf-secret-copy-job
 
+  cosign:
+    BASE_HOSTNAME: apps.<base_domain>
```

Comment:
can you also add lines 55-56 in the examples/values-* files, it's easy to place these in the wrong section (I accidentally put these under the scaffold section and had to correct the deployment the first time) - so like 
```
configs:
  cosign:
    BASE_HOSTNAME: apps.rosa.gwk4p-hx5su-j45.6hoh.p3.openshiftapps.com
```

Reply:
Nvm, I answered my own question when testing hahah, good idea I was trying to do something like this but never thought of doing that :)